src/schedule/ucc_schedule.c

```diff
@@ -119,6 +119,10 @@ ucc_status_t ucc_coll_task_init(ucc_coll_task_t *task,
     task->post                 = ucc_dummy_post;
     task->finalize             = ucc_dummy_finalize;
     task->progress             = ucc_dummy_progress;
+
+    // Prevent asymmetric memory copy-out of garbage address at task complete
+    task->bargs.asymmetric_save_info.scratch = NULL;
```

Comment:
Should this be aligned with the previous assignments, even though I added a comment and some spacing?

Reply:
I think it looks cleaner if the majority are aligned, and the one off, whose nested elements would skew everything, is best left as the outlier, just my 2c